<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StackedWidget</class>
 <widget class="QStackedWidget" name="StackedWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StackedWidget</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="homePage">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item alignment="Qt::AlignHCenter|Qt::AlignTop">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../resources/resources.qrc">:/cinves-logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter|Qt::AlignTop">
     <widget class="QLabel" name="lblHome">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>850</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>900</width>
        <height>500</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600; color:#888a85;&quot;&gt;AbinDebugger:&lt;/span&gt;&lt;span style=&quot; font-size:20pt; color:#888a85;&quot;&gt; Automatic Debugging of Semantic Bugs in Python Programs Using Abductive Inference&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="testSuitePage">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="5" column="0" alignment="Qt::AlignTop">
     <widget class="QPushButton" name="btnLoadTestSuite">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Load Test Suite</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="Line" name="line">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTableWidget" name="tableTypes">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>256</width>
        <height>400</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>input_args</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>type</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTableWidget" name="tableTestSuite">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>691</width>
        <height>400</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>test_case</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>input_args</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="3" column="1" alignment="Qt::AlignHCenter|Qt::AlignTop">
     <widget class="QLabel" name="lblTestSuite">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Test Suite</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0" alignment="Qt::AlignHCenter|Qt::AlignTop">
     <widget class="QLabel" name="lblTypes">
      <property name="text">
       <string>Test Arguments Types</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="abductionPage">
   <layout class="QGridLayout" name="gridLayout">
    <item row="9" column="0">
     <widget class="QLabel" name="lblLogging">
      <property name="text">
       <string>Logging</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QComboBox" name="cmbTargetFunction">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="3">
     <widget class="QPushButton" name="btnStartDebug">
      <property name="text">
       <string>Start Debugging</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" rowspan="2">
     <widget class="QFrame" name="frmAbductionChart">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(243, 243, 243);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
    </item>
    <item row="1" column="1" rowspan="8">
     <widget class="Line" name="vLineLogging">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="8">
     <widget class="QPlainTextEdit" name="txtLogging">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(243, 243, 243);</string>
      </property>
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::NoWrap</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="4">
     <widget class="Line" name="hLineLogging">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="lblComplexity">
      <property name="text">
       <string>Complexity</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QSpinBox" name="snbComplexity">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="value">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QListWidget" name="lstModel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(243, 243, 243);</string>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="lblModel">
      <property name="text">
       <string>Current Model</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QLabel" name="lblTargetFunction">
      <property name="text">
       <string>Target&lt;br&gt;Function</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="lblAbductionChart">
      <property name="text">
       <string>Abduction Complexity Chart</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="testPage"/>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
